src/knex-store.ts

```diff
@@ -166,6 +166,17 @@ function knex_store(this: any, options: Options) {
     }
   )
 
+  seneca.add(
+    'sys:entity,transaction:adopt',
+    async function (msg: any, reply: any, handle: any) {
+      console.log('ADOPT', handle)
+      const trxProvider = handle
```

Comment:
`handle` comes from the message:
https://github.com/senecajs/seneca-entity/blob/master/src/entity.ts#L313

Reply:
Yep, done.